### PR TITLE
Audit/admin

### PR DESCRIPTION
**Pull request by antobrines**

### CODE REVIEW

File: src/controllers/admin.controller.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+});
+
+
+const removeComment = catchAsync(async (req, res, next) => {
+  const {
+    id
+  } = req.params;
+  const user = req.user;
+  const comment = await commentService.remove(id, user.userId, true);
+  successF('Comment Deleted', comment, httpStatus.OK, res, next);
```
Comment (defless) [inline src/controllers/admin.controller.js:32]: Maybe an empty body would be better and more logical for a delete response ? 
I found no strict rules about this, so I think it's as we want.

File: src/validations/comment.validation.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  body: Joi.object().keys({
+    content: Joi.string().required(),
+  }),
+  params: Joi.object().keys({
+    id: Joi.string().required(),
+  })
+};
+
+const remove = {
+  body: Joi.object().keys({}),
```
Comment (defless) [inline src/validations/comment.validation.js:20]: Why this empty conditions ?

File: src/services/comment.service.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  placeHolder: 'x',
+  emptyList: true
+});
+badWords.addWords(...frenchBadwordsList.array);
+
+const create = async (contentBody) => {
+  if (badWords.isProfane(contentBody.content)) {
+    throw new Error('Your comment contains bad words');
+  }
+  return Comment.create(contentBody);
```
[the remark this answers is not in the document]
Comment (antobrines) [reply src/services/comment.service.js]: Missing, I added them

File: src/validations/comment.validation.js (hunk 2 above)
Comment (antobrines) [reply src/validations/comment.validation.js:20]: To block any body

**Pull request merged**